File: BlogWebAPI/BlogWebAPI/Entities/Blog.cs
namespace BlogWebAPI.Entities
{
    public class Blog
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public int IdUser { get; set; }
        public User? User { get; set; }
        public ICollection<Article>? Articles { get; set; } = null!;
    }
}

File: angular-blog/src/app/models/blog.ts
import { Article } from "./article";
import { User } from "./user";


export class Blog{
    
    
    // constructor( id:number, name:string, idUser:number, user:User,articles?:Article[]) {
    //     this.id=id;
    //     this.name=name;
    //     this.idUser=idUser;
    //     this.user=user;
    //     this.articles = articles;
        
    // }
    id?:number;
    name?:string;
    idUser?:number;
    user?: User;
    articles?: Article[];
}